openmdao_book/features/building_blocks/solvers/broyden.ipynb

```diff
@@ -50,7 +50,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The BroydenSolver also contains a slot for a linear solver and a slot for a linesearch. See the :ref:`feature doc about linesearches <feature_line_search>` for more about these.\n",
+    "The BroydenSolver also contains a slot for a linear solver and a slot for a linesearch. See the [linesearch section](linesearch-section) for more about these.\n",
```

Comment:
Looks like the linked worked but not sure it zoomed to the header in question. I believe it needs a `#` in front